selftests/checkall

```diff
@@ -85,6 +85,18 @@ parallel_selftests() {
                 for FAILURE in $(cat "${TMPS[$I]}" | sed -n 's/\(ERROR\|FAIL\): \([^ ]*\) (\([^)]*\)).*/\3.\2/p'); do
                     FAILED_ONCE+=("$FAILURE")
                 done
+                # On Python 3.4, load errors are not treated as test failures, and we can
+                # not easily tell which test failed to be loaded.  Let's return immediately.
```

Comment:
Thanks, this is better. Although I thought about it a bit and basically this is "system overload" issue. How about killing the remaining processes and running `selftest/checkall` instead of exit? That should decrease the need for restarting jobs in Travis (or manual selfchecks, although I use py3.6).

Reply:
@ldoktor can you elaborate on your affirmation that this is a "system overload" issue? While I can not prove that a system under heavy load will fail to load a test, this is clearly reproducible by having unloadable modules (that was the original issue with `StringIO` being used on Python 3).

Reply:
Hmm, I thought this was Travis issue where it failed to find the module even though it was there. Thank you for clearing this out and let me re-check and merge it.